src/helpers.js

```diff
@@ -151,7 +149,7 @@ export const findCellRectClosestToPos = $pos => {
   if (cell) {
     const table = findTableClosestToPos($pos);
     const map = TableMap.get(table.node);
-    const cellPos = cell.pos - table.pos - 1;
+    const cellPos = cell.pos - table.start;
```

Comment:
was this a typo? it looks like it was meant to be `cell.pos - table.pos`

Reply:
that's a funny one, table map class contains positions relative to the table (not absolute positions). Since `cell.pos` is an absolute position (relative to `doc`), we need to remap cell pos.